docs/changelogs/v0.22.md

```diff
@@ -23,6 +24,17 @@ to V2 only in the future.
 
 **TODO**: add links to IPIP https://github.com/ipfs/specs/issues/376
 
+# go-libp2p v0.29.0 update
+
+We updated from [go-libp2p](https://github.com/libp2p/go-libp2p) [v0.27.7](https://github.com/libp2p/go-libp2p/releases/tag/v0.27.7) to [v0.29.0](https://github.com/libp2p/go-libp2p/releases/tag/v0.29.0).
+
+The main intresting new feature is smart dialling, this is a prioritisation algorithm that will try to rank addresses and protocols.
+So when we are contacting one peer we aren't firing ~4 connections at attempts in parallel anymore.
+
+This includes a breaking change to `ipfs id` and some of the `ipfs swarm` commands, we do not report `ProtocolVersion` anymore.
+
+This used to be harcoded `ipfs/0.1.0` and sent to other peers but never did anything intresting.
```

Comment:
Some suggested language adjustment below.

A couple of things:
1. Is it because of the go-libp2p that this "protocol version" aspect was dropped.  Does go-libp2p no longer support this?
2. We still have a way to identify Kubo noes and their version right?
3. Given this is a "breaking change" lets also call it out at the top of the changelog as well (and can just link to this go-libp2p section for details?)

```suggestion
# go-libp2p v0.29.0 update with smart dialing

We updated from [go-libp2p](https://github.com/libp2p/go-libp2p) [v0.27.7](https://github.com/libp2p/go-libp2p/releases/tag/v0.27.7) to [v0.29.0](https://github.com/libp2p/go-libp2p/releases/tag/v0.29.0).  This release includes smart dialing, which is a prioritization algorithm that will try to rank addresses and protocols rather than attempting all options in parallel.  Anecdotally, we have observed [Kubo nodes make 30% less dials](https://github.com/libp2p/go-libp2p/issues/2326#issuecomment-1644332863) with no to low latency impact.

This includes a breaking change to `ipfs id` and some of the `ipfs swarm` commands.  We no longer report `ProtocolVersion`.  This used to be hardcoded as `ipfs/0.1.0` and sent to other peers but was not providing any distinguishing value.
```

Reply:
See https://github.com/libp2p/go-libp2p/issues/2294 for more context regarding protocol version.